packages/browser-extension/src/utils/gapi.ts

```diff
@@ -40,6 +40,20 @@ export class GapiClient {
     }
   }
 
+  async addComment(
+    fileId: string,
+    content: string,
+  ): Promise<null> {
```

Comment:
Oh. I'm curious what will happen for a doc? (I mean, where will the comment be placed?)

Reply:
The comment is not visible on the document. But there should be an email notification.
There is a somewhat similar situation where the commented text gets deleted: that makes the comment not visible on the document even though the comment was never resolved.

In either case you can see the comments by clicking on "Open Comment History" in the upper right.

Reply:
I think I would prefer to place a visible comment on the doc. But this is more work because it requires selecting some text (probably the first line) and I would want to check that the text is not already selected for a different comment since overlapping comments can be difficult to deal with.